code/drasil-data/lib/Data/Drasil/Quantities/Physics.hs

```diff
@@ -44,82 +44,82 @@ acceleration, angularAccel, angularDisplacement, angularVelocity, chgInVelocity,
-chgMomentum          = uc CP.chgMomentum (Concat [cDelta,vec cP]) impulseU
-momentum             = uc CP.momentum (vec cP) impulseU
-moment               = uc CP.moment   (vec cM) torqueU
-moment2D             = uc CP.moment   cM       torqueU
+acceleration         = uc CP.acceleration (vec lA) (Vect Real) accelU
+angularAccel         = uc CP.angAccel lAlpha Real angAccelU
+angularDisplacement  = uc CP.angDisp lTheta Real radian
+angularFrequency     = uc CP.angFreq cOmega Real second
+angularVelocity      = uc CP.angVelo lOmega Real angVelU
+chgInVelocity        = uc CP.chgInVelocity (Atop Delta $ vec lV) Real velU
+constAccel           = uc CP.constAccel (sup lA lC) Real accelU
+displacement         = uc CP.displacement (vec lU) Real metre
+distance             = uc CP.distance lD Real metre
+energy               = uc CP.energy cE Real joule
+force                = uc CP.force (vec cF) Real newton
+frequency            = uc CP.frequency lF Real hertz
+gravitationalAccel   = uc CP.gravitationalAccel (vec lG) Real accelU
+gravitationalConst   = uc CP.gravitationalConst cG Real gravConstU
+height               = uc CP.height lH Real metre
+impulseS             = uc CP.impulseS lJ Real impulseU
+impulseV             = uc CP.impulseV (vec cJ) Real impulseU
+kEnergy              = uc CP.kEnergy (Concat [cK, cE]) Real joule
+linearAccel          = uc CP.linAccel (Concat [vec lA, label "(", lT, label ")"]) Real accelU
+linearDisplacement   = uc CP.linDisp  (Concat [vec lU, label "(", lT, label ")"]) Real metre
+linearVelocity       = uc CP.linVelo  (Concat [vec lV, label "(", lT, label ")"]) Real velU
+momentOfInertia      = uc CP.momentOfInertia (vec cI) Real momtInertU
+chgMomentum          = uc CP.chgMomentum (Concat [cDelta,vec cP]) Real impulseU
+momentum             = uc CP.momentum (vec cP) Real impulseU
+moment               = uc CP.moment   (vec cM) Real torqueU
+moment2D             = uc CP.moment   cM       Real torqueU
 -- FIXME: moment2D should eventually be a specialization of moment, not separately defined
-period               = uc CP.period cT second
-position             = uc CP.position (vec lP) metre
-positionVec          = uc CP.positionVec (vec lR) metre
-potEnergy            = uc CP.potEnergy (Concat [cP, cE]) joule
-pressure             = uc CP.pressure lP pascal
-speed                = uc CP.speed lV velU
-scalarAccel          = uc CP.scalarAccel lA accelU
-scalarPos            = uc CP.scalarPos lP metre
-tension              = uc CP.tension (vec cT) newton
-time                 = uc CP.time lT second
-torque               = uc CP.torque (vec lTau) torqueU
-velocity             = ucs' CP.velocity (vec lV) (Vect Real) velU
-weight               = uc CP.weight cW newton
-fOfGravity           = uc CP.fOfGravity (sub (vec cF) (vec lG)) newton
+period               = uc CP.period cT Real second
+position             = uc CP.position (vec lP) Real metre
+positionVec          = uc CP.positionVec (vec lR) Real metre
```

Comment:
So `positionVec` should probably be `Vec Real` and not `Real`! But doing that should be in a separate PR. Lots of things in here have `vec` for display and should probably have `Vec` type as well. But Drasil might not be ready for it, so it should be done incrementally.

Reply:
Issue created: https://github.com/JacquesCarette/Drasil/issues/3246 - the fact that we could more easily see this seems to imply that not assuming the `Space` was a good idea!